packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -2212,6 +2213,402 @@ curl -v -X GET \
 ]
 ```
 
+#### List apps by name
```

Comment:
the other examples seem to use `List applications` vs `List apps`

Reply:
@luishernandezcruz-okta - This was actually suggested by @jakubvul-okta. You can see his review above or this should take you directly to his suggestion - https://github.com/okta/okta-developer-docs/pull/2127#discussion_r632037113

Reply:
ahh i see. either works since we are pretty inconsistent with `app` vs `application` in these docs